### PR TITLE
[Magic Firewall] Enable ids dashbaord

### PR DESCRIPTION
**Pull request by marciocloudflare**
Closes PCX-1430

### CODE REVIEW

File: content/magic-firewall/how-to/enable-ids.md [hunk 1]
```diff
@@ -2,19 +2,32 @@
 title: Enable IDS
 pcx_content_type: how-to
 weight: 5
+meta:
+  title: Enable Intrusion Detection Systems (IDS)
 ---
 
-# Enable IDS
+# Enable Intrusion Detection Systems (IDS)
```
Comment (jason-cf) [inline content/magic-firewall/how-to/enable-ids.md]: systems or system?

File: content/magic-firewall/how-to/enable-ids.md [hunk 2]
```diff
@@ -2,19 +2,32 @@
...
-## Create a new ruleset
 
-Follow instructions in the [Rulesets Engine Page](/ruleset-engine/basic-operations/view-rulesets/) to view all rulesets for your account. You must see a ruleset with phase `magic-transit-ids-managed` and kind `managed`. If not, please contact your account team. The managed ruleset ID will be used in the next step.
+{{<tabs labels="Dashboard | API">}}
+{{<tab label="dashboard" no-code="true">}}
 
-Next, create a root ruleset with a single rule in the `magic_transit_ids_managed` phase by running:
+1. Log in to your [Cloudflare dashboard](https://dash.cloudflare.com/login), and select your account.
+2. Select **Magic Firewall** > **IDS**.
+3. Enable IDS.
```
Comment (jason-cf) [inline content/magic-firewall/how-to/enable-ids.md:21]: consider changing to `toggle` as this is the word that comes to mind w/ the on/off mechanism in dash

File: content/magic-firewall/how-to/enable-ids.md [hunk 3]
```diff
@@ -64,4 +73,11 @@ https://api.cloudflare.com/client/v4/accounts/{account_id}/rulesets/{root_rulese
...
-Similarly, sending a patch request with the `enabled` field set to `true` will enable IDS.
\ No newline at end of file
+Similarly, sending a patch request with the `enabled` field set to `true` will enable IDS.
+ 
+{{</tab>}}
+{{</tabs>}}
+
+## Next steps
+
+You must configure Logpush to log detected risks. Refer to  [Configure a Logpush destination](/magic-firewall/how-to/use-logpush-with-ids/) for more information. Additionally, all traffic that is analyzed can be accessed via [network analytics](/analytics/network-analytics/). Refer to [GraphQL Analytics](/magic-firewall/tutorials/graphql-analytics/) to query the analytics data.
```
Comment (jason-cf) [inline content/magic-firewall/how-to/enable-ids.md]:
```suggestion
You must configure Logpush to log detected risks. Refer to [Configure a Logpush destination](/magic-firewall/how-to/use-logpush-with-ids/) for more information. Additionally, all traffic that is analyzed can be accessed via [network analytics](/analytics/network-analytics/). Refer to [GraphQL Analytics](/magic-firewall/tutorials/graphql-analytics/) to query the analytics data.
```

**Review by jason-cf — APPROVED** [review]

File: content/magic-firewall/how-to/enable-ids.md (hunk 2 above)
Comment (marciocloudflare) [reply content/magic-firewall/how-to/enable-ids.md:21]: It's a good call, although we decided against that line of writing in one of our content critiques

**Pull request merged**